### PR TITLE
Addition of Vegetarian Nachos

### PR DESCRIPTION
**Pull request by raithwind**
Summary

SUMMARY: Content "Adding Vegetarian Nachos, and Cheesy Vegetarian Nachos"

Purpose of change

Adding vegetarian options for Nachos as per the issue #41566 

Describe the solution

Add 2 item, and corresponding recipes. 

Describe alternatives you've considered

Not doing it and letting someone else do it instead.

Testing

Tested by loading a world, debug spawning the new items, as well as debug increasing stats to craft them and debug spawning all the requirements. Was spawnable, craftable and edible.

Additional context

None

### CODE REVIEW

**Comment by Fris0uman** [comment]
You need to astyle your changes, see: https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_STYLE.md

**Comment by raithwind** [comment]
Thanks @Fris0uman, I have used the linter now on both files modified and should now pass. I can't seem to get the PR validator to like my PR comment though.

**Comment by Fris0uman** [comment]
Also you need to remove the [] from the summary line
`SUMMARY: Content: "Adding Vegetarian Nachos, and Cheesy Vegetarian Nachos"`

**Comment by raithwind** [comment]
Thanks, got it fixed now. All the parts that I have changed have now passed the autovalidations. Now just for people to assess the changes and decide whether to give it the go ahead.

File: data/json/items/comestibles/junkfood.json [hunk 1]
```diff
@@ -743,6 +743,53 @@
...
+    "weight": "22 g",
+    "color": "yellow",
+    "spoils_in": "20 hours",
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "charges": 3,
+    "quench": -10,
+    "calories": 348,
+    "description": "Salted chips made from corn tortillas with beans and smothered in cheese. Delicious, even if you're not a vegetarian.",
```
Comment (Fris0uman) [inline data/json/items/comestibles/junkfood.json:783]:
```suggestion
    "description": "Salted chips made from corn tortillas with beans and smothered in cheese.  Delicious, even if you're not a vegetarian.",
```

File: data/json/items/comestibles/junkfood.json [hunk 2]
```diff
@@ -743,6 +743,53 @@
...
+    "copy-from": "nachos",
+    "name": "vegetarian nachos with cheese",
+    "weight": "22 g",
+    "color": "yellow",
+    "spoils_in": "20 hours",
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "charges": 3,
+    "quench": -10,
```
Comment (Fris0uman) [inline data/json/items/comestibles/junkfood.json:781]:
```suggestion
```
you're not changing the value so you don't need to have that field.

File: data/json/items/comestibles/junkfood.json [hunk 3]
```diff
@@ -743,6 +743,53 @@
...
+  {
+    "type": "COMESTIBLE",
+    "id": "nachosvc",
+    "copy-from": "nachos",
+    "name": "vegetarian nachos with cheese",
+    "weight": "22 g",
+    "color": "yellow",
+    "spoils_in": "20 hours",
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
```
Comment (Fris0uman) [inline data/json/items/comestibles/junkfood.json:778]:
```suggestion
```

File: data/json/items/comestibles/junkfood.json [hunk 4]
```diff
@@ -743,6 +743,53 @@
...
+    "color": "yellow",
+    "quench": -10,
+    "calories": 228,
+    "description": "Salted chips made from corn tortillas, now with beans.  Could probably use some cheese, though.",
+    "price": 250,
+    "weight": "125 g",
+    "spoils_in": "1 day",
+    "price_postapoc": 600,
+    "container": "bag_plastic",
+    "comestible_type": "FOOD",
```
Comment (Fris0uman) [inline data/json/items/comestibles/junkfood.json:761]:
```suggestion
```

File: data/json/items/comestibles/junkfood.json [hunk 5]
```diff
@@ -743,6 +743,53 @@
...
     "vitamins": [ [ "vitA", 10 ], [ "calcium", 10 ], [ "iron", 12 ], [ "vitB", 29 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "nachosv",
+    "copy-from": "nachos",
+    "name": "vegetarian nachos",
+    "symbol": "%",
+    "color": "yellow",
+    "quench": -10,
```
Comment (Fris0uman) [inline data/json/items/comestibles/junkfood.json:753]:
```suggestion
```

**Comment by raithwind** [comment]
Thanks for the edits @Fris0uman it's the first time working with inheritance, and I was using the code for the meat equivalents as a base for the code.

**Comment by Fris0uman** [comment]
Reciep test says:
```
RecipeID: nachosvc, default is 1044 Calories,
Current recipe range: 1275-1321, Average 1301
Average recipe Calories must fall within this range, derived from default Calories: 835-1253
```
So you probably need to edit the calorie of nachosvc

**Comment by raithwind** [comment]
Thanks for your patience with me @Fris0uman hopefully the recipe should now fall within reasonable range.

**Pull request merged**